GDJS/Runtime/pixi-renderers/layer-pixi-renderer.ts

```diff
@@ -221,7 +217,32 @@ namespace gdjs {
             this._threePlaneTexture.magFilter = filter;
             this._threePlaneTexture.wrapS = THREE.ClampToEdgeWrapping;
             this._threePlaneTexture.wrapT = THREE.ClampToEdgeWrapping;
-            this._threePlaneMaterial.map = this._threePlaneTexture;
+            const noGammaCorrectionShader: THREE.ShaderMaterialParameters = {
```

Comment:
Add a comment (see my other comment) that we don't want a double gamma correction, so we disable the one done by Three